Applications/Utils/Tests.cmake

```diff
@@ -69,29 +69,41 @@ AddTest(
 )
 
 AddTest(
-    NAME identifySubdomains_riverTriangleMesh
+    NAME identifySubdomains_PntLineSFC_DomainTRIMesh
```

Comment:
PointLineSurface would be more readable than mixing three types of naming.
I'd call the whole thing `identifySubdomains_pointLineSurfaceOnTriangleMesh`.